modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -80,13 +80,19 @@ This option requires an Enterprise license.
 This mode is not recommended for production clusters. Only set to `off` if you need to move partitions manually.
 |===
 
-== Partition balancing with Kafka API
+== Manual partition balancing
```

Comment:
```suggestion
== Manually move partitions
```
@daisukebe what do you think about changing this heading to _Manually move partitions_ to align it with the heading earlier on this page _Manually change leadership_?

Reply:
Totally makes sense!